src/css/brandColors.test.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable n/no-sync */
+/* eslint-disable jsdoc/require-jsdoc */
+import path from 'path';
+
+import { getCssColorVariables } from '../../utils';
+import jsonData from '../figma/brandColors.json';
+
+describe('CSS Variables', () => {
+  const cssFilePath = path.join(__dirname, 'generated-brand-colors.css');
```

Comment:
Can we test the current brand color styles sheet here?
```suggestion
  const cssFilePath = path.join(__dirname, 'brand-colors.css');
```